src/probe_modules/module_dns.c

```diff
@@ -81,14 +81,18 @@ static int num_ports;
 
 const char default_domain[] = "www.google.com";
 const uint16_t default_qtype = DNS_QTYPE_A;
+const uint8_t default_rdbit = 0xFF;
```

Comment:
Shouldn't this be `0x01` since the default value is 1, not 255? Or does any non-zero value have the same effect when processed by the probed target.

Reply:
Update: just noticed that this is used to set a bit-value, so this is fine. Approving